.github/PULL_REQUEST_TEMPLATE/app_update.md

```diff
@@ -0,0 +1,63 @@
+# PR checklist to update an application already in the New Relic One Catalog
+
+## Nerdpack Approver
+
+- Provide the **GitHub handle** for the final Approver of this Nerdpack in your Pull Request. *Note: this is the person who will need to provide explicit sign off for the change ahead of deployment.*
+- If known, what is the requested date for the deployment of this functionality? *Note: this information is regarded as a courtesy. Neither New Relic nor the **nr1-catalog maintainer team** are under any obligation to meet this request. It simply provides a useful target for completion.*
+
+### Functional Changes
+
+List the functional changes in the pull request.
+
+- Major feature #1
+- Major feature #2
+
+*Note: The rest of this template is a checklist and guide for ensuring that your pull request is swiftly and consistently approved.*
+
+## Basic Nerdpack Validation
+
+Validate that the following are present in your PR ahead of submitting:
+
+- [ ] Your application's reference in `.gitmodules` file is updated to the check-in your expect
```

Comment:
Tiny correction - the submodule reference doesn't change in `.gitmodules` when an update occurs. Git's internal object store is updated with the new submodule SHA. The only way to "see" that change is via `git submodule status`. 

```suggestion
- [ ] Your application's submodule commit reference (SHA-hash value) has been updated to the value you expect (validate via `git submodule status`)
```